tests/unittest/_torch/visual_gen/test_model_loader.py

```
@@ -480,3 +480,139 @@ def test_fp8_vs_bf16_memory_comparison(checkpoint_exists):
+# =============================================================================
+
+
+def test_load_diffusion_quant_config_per_layer_parsing():
+    """Test that per_layer overrides are parsed into a Dict[pattern, QuantConfig]."""
+    from tensorrt_llm._torch.visual_gen.config import DiffusionModelConfig
+    from tensorrt_llm.quantization.mode import QuantAlgo
+
+    parse = DiffusionModelConfig.load_diffusion_quant_config
+
+    # Mixed: global NVFP4, attention layers override to FP8_BLOCK_SCALES.
+    qc, layer_cfg, dwq, daq = parse(
```

Ruff (0.15.17)

[warning] Unpacked variable `daq` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)